index.html

```diff
@@ -15,68 +15,115 @@
+        <div class="column0">
+          <img src="/images/image-computer.png" alt="">
+        </div>
+        <div class="column0">
+        <h3>Quick Search</h3>
+        <P>Easily search your snippets by content, category, web address, application, and more.</P>
+      
+        <h3>iCloud Sync</h3>
+        <p>Instantly saves and syncs snippets across all your devices.</p>
+      
+        <h3>Complete History</h3>
+        <p>Retrieve any snippets from the first moment you started using the app.</p> 
+      </div>  
+    </div>
+    <h2>Access Clipboard anywhere</h2>
+    <p>Whether you’re on the go, or at your computer, you can access all your Clipboard 
+          snippets in a few simple clicks.
+    </p>
+    <img src="/images/image-devices.png" alt="">
+    <h2>Supercharge your workflow</h2>
+    <p>We’ve got the tools to boost your productivity.</p>
+    <div class="second-row">
+      <div class="column1"></div>
+        <img src="images/icon-blacklist.svg">
+        <h2>Create blacklists</h2>
+        <p>Ensure sensitive information never makes its way to your clipboard by excluding certain sources.</p>
+      </div>
+      <div class="column1">
+        <img src="images/icon-text.svg">
+        <h2>Plain text snippets</h2>
+        <p>Remove unwanted formatting from copied text for a consistent look.</p>
+      </div>
+      <div class="column1">
+        <img src="images/icon-preview.svg">
+        <h2>Sneak preview</h2>
+        <p>Quick preview of all snippets on your Clipboard for easy access.</p>
+      </div>
+    </div>
+    <section>
+      <div class="third-row">
+        <div class="column2">
+          <img src="images/logo-google.png">
+        </div>
+        <div class="column2">
+          <img src="images/logo-ibm.png">
+        </div>
+        <div class="column2">
+          <img src="images/logo-microsoft.png">
+        </div>
+        <div class="column2">
+          <img src="images/logo-hp.png">
+        </div>
+        <div class="column2">
+          <img src="images/logo-vector-graphics.png">
+        </div>
+      </div>
+    </section>
+
+  </section>
+  <section>
+    <h2>Clipboard for iOS and Mac OS</h2>
+    <p>Available for free on the App Store. Download for Mac or iOS, sync with iCloud 
+      and you’re ready to start adding to your clipboard.
+    </p>
+    <button>Download for iOS</button>
+    <button>Download for Mac</button>
+  </section>
+  <hr />
+  <div class="fourth-row">
+    <div class="column3">
+      <img src="/images/logo.svg">
+    </div>
+    <div class="column3">
+      <P>FAQs</P>
+      <p>Contact Us</p>
+    </div>
+    <div class="column3">
+      <p>Privacy Policy</p>
+      <p>Press Kit</p>
+    </div>
+    <div class="column3">
+      <p>Install Guide</p>
+    </div>
+    <div class="column3">
+      <img src="/images/icon-facebook.svg">
+      <img src="/images/icon-twitter.svg">
+      <img src="/images/icon-instagram.svg">
+    </div>
+  </div>
```

Comment:
You did really good.
I'll send you a copy of the one we'll be using. copy and paste it in your index.html and make another pull request. You should also go through it and see how it differs from your own.